setup.py

```diff
@@ -8,27 +8,27 @@
 
 # To update the package version number, edit DeepRank-GNN/__version__.py
 version = {}
-with open(os.path.join(here, 'deeprank_gnn', '__version__.py')) as f:
+with open(os.path.join(here, 'deeprank_gnn_2', '__version__.py')) as f:
     exec(f.read(), version)
 
 with open('README.md') as readme_file:
     readme = readme_file.read()
 
 setup(
-    name='DeepRank-GNN',
+    name='DeepRank-GNN-2',
     version=version['__version__'],
-    description='Graph Neural network Scoring of protein-protein conformations',
+    description='TBD',
```

Comment:
I would just leave the current description, probably seems suitable for deeprank-gnn-2 as well.